lms/djangoapps/grades/tasks.py

```diff
@@ -189,7 +188,8 @@ def _update_subsection_grades(
                         subsection_grade=subsection_grade,
                     )
 
-        except DatabaseError as exc:
+        except Exception as exc:   # pylint: disable=broad-except
+            log.info("_update_subsection_grades unexpected failure, retrying. Error: {}".format(repr(exc)))
             raise _retry_recalculate_subsection_grade(
```

Comment:
3, we don't touch the celery-defined default value: http://docs.celeryproject.org/en/latest/userguide/tasks.html#Task.max_retries